src/main/java/org/n52/iceland/ogc/filter/FilterConstants.java

```diff
@@ -211,7 +255,38 @@ public static TimeOperator2 from(TimeOperator to) {
         PropertyIsLike,
         PropertyIsNil,
         PropertyIsNull,
-        PropertyIsBetween
+        PropertyIsBetween;
+
+        public static String asString(ComparisonOperator co) {
+            switch (co) {
+                case PropertyIsEqualTo:
+                    return "EqualTo";
+                case PropertyIsNotEqualTo:
+                    return "NotEqualTo";
+                case PropertyIsLessThan:
+                    return "LessThan";
+                case PropertyIsGreaterThan:
+                    return "GreaterThan";
+                case PropertyIsLessThanOrEqualTo:
+                    return "LessThanOrEqualTo";
+                case PropertyIsGreaterThanOrEqualTo:
+                    return "GreaterThanEqualTo";
+                case PropertyIsLike:
+                    return "Like";
+                case PropertyIsNil:
+                    return "Nil";
+                case PropertyIsNull:
+                    return "NullCheck";
+                case PropertyIsBetween:
+                    return "Between";
+                default:
+                    throw new IllegalArgumentException(String.format("Operators %s is not supported.", co));
```

Comment:
Good Idea - will I not break somebody code with that maybe? Because I change the ``toString(..)`` method.